1.1.0/shipment-status.md

```diff
@@ -500,7 +500,7 @@ If all goes well, the company **Pulp** will receive a response like this:
         }
       ],
       "shipmentStatus": "Active",
-      "shipmentArrivalStatus": "OnTime",
```

Comment:
This change needs to be synchronized with the mock file `1.1.0/mock/samples/shipment-use-case.B.step-2.json`.